README.md

```diff
@@ -487,6 +487,16 @@ end
 
 ```
 
+### Development
+
+You can get development environment. It requires [vagrant](https://www.vagrantup.com/).
```

Comment:
This gem doesn't require vagrant for development. Can you remove the vagrant stuff from the PR?